homeassistant/components/isy994/services.py

```diff
@@ -332,6 +332,13 @@ def async_cleanup_registry_entries(service: ServiceCall) -> None:
 
     async def async_reload_config_entries(service: ServiceCall) -> None:
         """Trigger a reload of all ISY config entries."""
+        async_log_deprecated_service_call(
+            hass,
+            call=service,
+            alternate_service="homeassistant.reload_core_config",
+            alternate_target=None,
```

Comment:
Should we give them the config entry id?

Reply:
I thought about but if they have more than 1 ISY/Polisy/eisy connected then you would have to raise an issue and log a warning for each config. Doable, but since that service lets you pick any device, entity, or the config, I omitted it.

Open to either option.

Reply:
Good call. Its not worth the trouble for the minute value I think it would provide.